### PR TITLE
Use \since instead of \note added in

### PR DESCRIPTION
**Pull request by m-kuhn**
Because information is good but semantics is better.

    find . -name '*.h' | xargs sed -i 's/[@\]note [Aa]dded .* \([0-9]*\.[0-9]*\)/\\since QGIS \1/'

sip test will still need to be adjusted

### CODE REVIEW

File: python/core/qgssettings.sip [hunk 1]
```diff
@@ -36,7 +36,9 @@ class QgsSettings : QObject
   - Misc
   - Auth
 
-.. versionadded:: 3
+.. note::
```
Comment (nyalldawson) [inline python/core/qgssettings.sip]: Looks like your regex needs to handle "3" and "2.14.7" format version numbers

**Comment by nyalldawson** [comment]
+1. Also a good move since it avoids the multiple different variations of these strings used too!

**Comment by nyalldawson** [comment]
Btw - you'll probably need to update the doxygen unit test as that checks to make sure new classes have an "added in" note.

**Comment by m-kuhn** [comment]
Not to forget about `Added in QGIS v2.4`.

**Comment by nyalldawson** [comment]
Should this be outputing "\since QGIS 2.4" or just "\since 2.4"? I'd lean toward just the version number

**Comment by nyalldawson** [comment]
Also when this lands it's time for @note/@param/@return to become \note \param \return ;)

**Comment by m-kuhn** [comment]
I was going for `QGIS 2.4` because branding and `QwtPolar x.y` etc. But can be plain numbers as well

**Comment by nyalldawson** [comment]
Love it!

**Pull request merged**